pyslim/slim_tree_sequence.py

```diff
@@ -461,6 +462,7 @@ def recapitate(self,
         else:
             ts = self
 
+        ts.dump("temp.trees")
```

Comment:
Did you mean to do this?

Reply:
no! thanks for the catch. You'll notice a force-push to `main` shortly after this PR was merged, so it's all good now: https://github.com/tskit-dev/pyslim/blob/0e4f0a187c7bc75c68a78593f541675d28e9c1a3/pyslim/slim_tree_sequence.py#L465